ai_tools/elasticgraph-mcp-server/pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[project]
+name = "elasticgraph-mcp-server"
+version = "0.1.1"
+description = "MCP Server for ElasticGraph to allow goose, Cursor, etc to create and manage your ElasticGraph project."
+readme = "README.md"
+requires-python = ">=3.12"
+license = "MIT"
+authors = [
+    { name = "Brian Sigafoos", email = "briansigafoos@block.xyz" },
+]
+keywords = ["elasticgraph", "mcp", "cursor", "windsurf", "model-context-protocol", "claude", "cline"]
+classifiers = [
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3.12",
+]
+dependencies = [
+    "httpx>=0.28.1",
+    "mcp[cli]>=1.3.0",
+    "packaging>=23.0",
+]
+
+[project.scripts]
+elasticgraph-mcp-server = "elasticgraph_mcp:main"
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build]
+packages = ["."]
+package-dir = "src"
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/elasticgraph_mcp"]
+
+[dependency-groups]
+dev = [
+    "ruff>=0.9.9",
+    "pytest>=8.0.0",
+    "pytest-cov>=4.1.0",
+    "pre-commit>=3.6.0",
+    "pytest-asyncio>=0.25.3",
```

Comment:
(Similar comment here about this possibly picking up new major versions with breaking changes).